api/types/accesslist/accesslist.go

```diff
@@ -21,6 +21,8 @@ import (
 	"fmt"
 	"time"
 
+	xreflect "golang.design/x/reflect"
```

Comment:
Should we use a library that mention in the docs?

> Warning: Not largely tested. Use it with care.

https://pkg.go.dev/golang.design/x/reflect#section-readme

Reply:
We considered other options for resolving our non-gogo issues. I read through all of the source code and it looks good. In addition, it's an official Go proposal that will likely be upstreamed.

Reply:
After discussing with Alan. We're gonna use `utils.StrictStructToObject` in the meantime while we figure out our cloning story for non-gogo resources.

Reply:
Thanks. We can follow up by making StrictStructToObject faster, if that's what we want.